web/app.js

```diff
@@ -254,6 +258,25 @@ var PDFViewerApplication = {
     ]).catch(function (reason) { });
   },
 
+  _initializeL10n() {
+    // Locale can be changed only when special debugging flags is present in
+    // the hash section of the URL, or development version of viewer is used.
+    // It is not possible to change locale for Firefox extension builds.
+    if (typeof PDFJSDev === 'undefined' || !PDFJSDev.test('PRODUCTION') ||
+        (!PDFJSDev.test('FIREFOX || MOZCENTRAL') &&
+         PDFViewerApplication.viewerPrefs['pdfBugEnabled'])) {
```

Comment:
This can be `this.viewerPrefs['pdfBugEnabled']`, given the current scope here (and you're already using `this.externalServices` below).